src/assets/templates/logs.template.html

```diff
@@ -1,775 +1,944 @@
+            const dropDownIcon = this;
+            const content = this.parentNode.parentNode.nextElementSibling;
+            if (content.style.maxHeight) {
+                content.style.maxHeight = null;
+                dropDownIcon.style.transform = "rotate(180deg)";
+                // editButton.style.visibility = "hidden";
+                // deleteButton.style.visibility = "hidden";
+                // shareButton.style.transform = "translate(40px, -18px)";
+                // shareButton.style.left = "-15px";
+            } else {
+                content.style.maxHeight = content.scrollHeight + "px";
+                dropDownIcon.style.transform = "rotate(0deg)";
+                // editButton.style.visibility = "visible";
+                // deleteButton.style.visibility = "visible";
+                // shareButton.style.transform = "translate(-48px, -18px)";
+                // shareButton.style.left = "0px";
+            }
+        });
+    }
+
+    /**
+    const deleteButtons = document.getElementsByClassName("cardHeaderDeleteLogButton");
+
+    for (let i = 0; i < deleteButtons.length; i++) {
+        deleteButtons[i].addEventListener("click", arg => {
+            if (arg && arg.currentTarget) {
+                const date_val = arg.currentTarget.getAttribute("data-val");
+                if (date_val) {
+                    const unix_date = date_val / 1000;
+                    vscode.postMessage({ command: "deleteLog", value: parseInt(unix_date, 10) });
+                }
+            }
+        });
+    }
+    **/
+
+    var days = [];
+
+    // expand the 1st card in the list
+    const logCards = document.getElementsByClassName("logCard");
+    if (logCards && logCards.length) {
+        // HTMLDivElement
+        const logCard = logCards[0];
+        const cardContent = logCard.getElementsByClassName("cardContent")[0];
+        cardContent.style.maxHeight = cardContent.scrollHeight + "px";
+
+        const shareButton = logCard.getElementsByClassName("cardHeaderShareButton")[0];
+        const editButton = logCard.getElementsByClassName("cardHeaderEditButton")[0];
+        // const deleteButton = logCard.getElementsByClassName("cardHeaderDeleteLogButton")[0];
+        const dropDownIcon = logCard.getElementsByClassName("cardHeaderDropDownButton")[0];
+        dropDownIcon.style.transform = "rotate(0deg)";
+        // editButton.style.visibility = "visible";
+        // deleteButton.style.visibility = "visible";
+        // shareButton.style.transform = "translate(-48px, -18px)";
+        // shareButton.style.left = "0px";
+
+        // set the day numbers that are found
+        for (let i = 0; i < logCards.length; i++) {
+            const card = logCards[i];
+
+            const day = card.getAttribute("data-val");
+            days.push(day);
+        }
+    }
+
+    var editButtons = document.getElementsByClassName("cardHeaderEditButton");
```

Comment:
at some point, we could move this all out of the html file

Reply:
cool, ok